addons/refuel/functions/fnc_connectNozzleAction.sqf

```diff
@@ -0,0 +1,109 @@
+/*
+ * Author: GitHawk et.al
+ * Calculates a connection for tilting
+ * With code from ace_attach
```

Comment:
> Modularity.

Fully agreed, but sometimes it's better to just make it a requirement. If it's not a full copy it's of course fine.